projects/js-packages/videopress-core/jsconfig.json

```diff
@@ -0,0 +1,8 @@
+// Config for a JS-using project. If using typescript, rename this to "tsconfig.json".
+{
+	"compilerOptions": {
+		"jsx": "react"
+	},
+	// List all sources and source-containing subdirs.
+	"include": [ "./index.jsx", "./src" ]
```

Comment:
There are many TS files in VideoPress, so this probably needs to be `tsconfig.json`